src/components/mc/cuda/kernel/mc_cuda_reduce_ops.h

```diff
@@ -21,4 +24,22 @@
 #define DO_OP_PROD_HALF(_v1, _v2) (__float2half(__half2float(_v1) * __half2float(_v2)))
 #endif
 
+#if defined(__CUDA_BF16_TYPES_EXIST__)
```

Comment:
I think we agreed that CUDART_VERSION >= 11000 is equivalent to  __CUDA_BF16_TYPES_EXIST__, let's use only CUDART_VERSION >= 11000 everywhere